src/apps/backend/modules/notification/email_service.py

```diff
@@ -1,8 +1,28 @@
+from typing import Optional
+
+from modules.logger.logger import Logger
 from modules.notification.internals.sendgrid_service import SendGridService
+from modules.notification.internals.account_notification_preferences_reader import AccountNotificationPreferenceReader
 from modules.notification.types import SendEmailParams
 
 
 class EmailService:
     @staticmethod
-    def send_email(*, params: SendEmailParams) -> None:
+    def send_email_for_account(
+        *, account_id: Optional[str] = None, bypass_preferences: bool = False, params: SendEmailParams
```

Comment:
If method name is `send_email_for_account`, it is expected for `account_id` to be present.